Wikimate.php

```diff
@@ -852,16 +866,20 @@ public function setText($text, $section = null, $minor = false, $summary = null)
 			$data = array(
 				'titles' => $this->title,
 				'prop' => 'info',
-				'intoken' => 'edit',
+				'curtimestamp' => 1,
 			);
 
 			$r = $this->wikimate->query($data);
 
-			$page = array_pop($r['query']['pages']); // Get the page
-
-			$this->starttimestamp = $page['starttimestamp']; // Update the starttimestamp
+			// Check for errors
```

Comment:
This adds error checking to the preceding query(), analog to such invocations elsewhere.

Reply:
Any reason why this comment was not added to the code itself?

Reply:
My remark should have been a (multiline) comment for new code lines 874 through 880. All other places where query() was called were followed by the code now also in the new lines. Not sure why it was previously missing, this was old code, but I don't think it's meaningful to spend more in-code comments on it.

Reply:
Got it, thanks for the context.